cli/config/config.go

```diff
@@ -46,6 +46,11 @@ func SetDir(dir string) {
 	configDir = dir
 }
 
+// Path returns the path to a file relative to the config dir
+func Path(p ...string) string {
+	return filepath.Join(append([]string{Dir()}, p...)...)
```

Comment:
For a follow-up; thinking if we should consider preventing directory traversal in this utility (i.e., prevent the resulting path from being outside of the `config` directory)